### DISCUSSION TITLE
Hugo cross-references and Dendron backlinking

### DISCUSSION THREAD

**Discussion by stradichenko** [dendronhq/dendron#3530] [RFCs/Ideas]
Is it possible to a string pattern into a functional backlink? Say [Text]({{< relref ".../folder/othernote" >}})
I work with an Hugo themed blog that links to other posts this way, but this pattern is not recognized in Dendron. Is there is a way to make it work with this format of cross reference?

https://gohugo.io/content-management/cross-references/

**Comment by kevinslin** [comment]
hmm, would this be different than creating a markdown link to the other note?

**Reply by stradichenko** [reply]
The problem is that Hugo don't see `[[]]` properly as a way of linking; nor does recognize typing `[postname](.../folder/post)` properly, and creates a faulty path `[postname](.../folder/currentpost/post)`instead... (I'm very new to Dendron so I'm still getting used to file-folder duality).
As I see it, this `relref` function does the same as markdown link, redirecting you to another internal post or document. What is missing is a way for Dendron to detect `relref` format as such and maybe offer markdowns with the same name and show them as connected in graph view; This would ease the use of Dendron for people to integrate it in their Hugo workflow.
